### PR TITLE
Updated Font-awesome from version 4.5.0 to version 4.6.3.

### PR DESCRIPTION
**Pull request by tim96**
Closes #

### Changed
- Updated Font-awesome from version 4.5.0 to version 4.6.3

### Subject

- Updated Font-awesome from version 4.5.0 to version 4.6.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,4 +6,5 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 ### Changed
-- Updated Bootstrap from version 3.3.5 to version 3.3.6
\ No newline at end of file
+- Updated Bootstrap from version 3.3.5 to version 3.3.6
```
Comment (greg0ire) [inline CHANGELOG.md]: Why does this line appear in the diff? I cannot see the difference…

Comment (tim96) [inline CHANGELOG.md]: @greg0ire Maybe `new line`?

Comment (greg0ire) [inline CHANGELOG.md]: Oh it might be because it is the last line of the file.

**Comment by greg0ire** [comment]
LGTM

**Comment by core23** [comment]
Thank you @tim96

**Comment by OskarStark** [comment]
Thank you @tim96 👍

**Pull request merged**